README.md

```diff
@@ -92,22 +92,31 @@ Now configure NGINX by adding this section:
     location = /web/session/logout {
         proxy_pass http://$authentication_provider_address:$authentication_provider_port/logout;
         proxy_pass_request_body off;
+        proxy_set_header Content-Length "";
     }
 
     location = /web/session/destroy {
         proxy_pass http://$authentication_provider_address:$authentication_provider_port/logout;
         proxy_pass_request_body off;
+        proxy_set_header Content-Length "";
     }
 
     location = /nginx-odoo-auth {
         proxy_pass http://$authentication_provider_address:$authentication_provider_port/auth;
         proxy_pass_request_body off;
+        proxy_set_header Content-Length "";
```

Comment:
The problem with Odoo not loading well was this, I changed it in the server's NGINX configuration and now it should work. Apparently it was waiting on a return that never came; strange that this did not surface in the Bottle program.

Reply:
This should also resolve the timeouts in the app.